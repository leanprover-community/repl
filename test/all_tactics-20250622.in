{"cmd": "example (P : Prop) (hp : P) : P := by\n  exact hp\n", "allTactics": true}

{"tactic": "exact hp", "proofState": 0}

