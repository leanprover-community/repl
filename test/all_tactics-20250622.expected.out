{"tactics":
 [{"usedConstants": [],
   "tactic": "exact hp",
   "proofState": 0,
   "pos": {"line": 2, "column": 2},
   "goals": "P : Prop\nhp : P\n‚ä¢ P",
   "endPos": {"line": 2, "column": 10}}],
 "env": 0}

{"proofStatus": "Completed", "proofState": 1, "goals": []}

